modules/e2ee/ManagedKeyHandler.js

```diff
@@ -167,6 +188,39 @@ export class ManagedKeyHandler extends KeyHandler {
         this.e2eeCtx.setKey(id, key, index);
     }
 
+    /**
+     * Handles the SAS ready event.
+     *
+     * @param {string} pId - The participant ID.
+     * @param {Uint8Array} sas - The bytes from sas.generate_bytes..
+     * @private
+     */
+    _onParticipantSasReady(pId, sas) {
+        this.conference.eventEmitter.emit(JitsiConferenceEvents.E2EE_VERIFICATION_READY, pId, sas);
+    }
+
+    /**
+     * Handles the sas available event.
+     *
+     * @param {string} pId - The participant ID.
+     * @private
+     */
+    _onParticipantSasKeyAvailable(pId) {
```

Comment:
nit: the name reads weird because it says "sas key", but it's in fact the fingerprint key. So let's just say "sas available"